source/clear-linux/guides/maintenance/download-verify-decompress-linux.rst

```diff
@@ -45,24 +45,28 @@ the screen followed by `OK`.
 For a more in-depth discussion of image verification including checking the
 certificate see :ref:`image-content-validation`.
 
-Uncompress the Clear Linux image
+.. incl-decompress-image:
+
+Decompress the Clear Linux image
```

Comment:
After use of |CL-ATTR|, subsequent use of "Clear Linux" can be replaced with |CL| sub.